packages/spatie-laravel-media-library-plugin/src/Tables/Columns/SpatieMediaLibraryImageColumn.php

```diff
@@ -63,13 +66,4 @@ public function getImagePath(): ?string
 
         return $record->getFirstMediaUrl($this->getCollection(), $this->getConversion());
     }
-
-    public function applyEagerLoading(Builder $query): Builder
-    {
-        if ($this->isHidden()) {
-            return $query;
-        }
-
-        return $query->with(['media']);
```

Comment:
Hi, I think this needs fixing before we merge this. Media needs to be eager-loaded as before.